metadata_service/proxy/atlas_proxy.py

```diff
@@ -416,9 +507,41 @@ def get_tags(self) -> List:
                 )
         return tags
 
-    def get_table_by_user_relation(self, *, user_email: str,
-                                   relation_type: UserResourceRel) -> Dict[str, Any]:
-        pass
+    def get_table_by_user_relation(self, *, user_email: str, relation_type: UserResourceRel) -> Dict[str, Any]:
+        params = {
+            'typeName': 'Reader',
+            'offset': '0',
+            'limit': '1000',  # Fixme (yesh) Pagination needs to be implemented.
+            'entityFilters': {
+                'condition': 'AND',
+                'criterion': [
+                    {
+                        'attributeName': self.QN_KEY,
+                        'operator': 'contains',
+                        'attributeValue': user_email
+                    },
+                    {
+                        'attributeName': self.BKMARKS_KEY,
+                        'operator': 'eq',
+                        'attributeValue': 'true'
+                    }
+                ]
+            },
+            'attributes': ['count', self.QN_KEY]
+        }
+        # Fetches the reader entities based on filters
+        search_results = self._driver.search_basic.create(data=params)
+
+        results = []
+        for record in search_results.entities:
+            res = self._parse_reader_qn(record.displayText)
+            results.append(PopularTable(
+                database='hive_table',
```

Comment:
why the database is hardocded as `hive_table` here ?

Reply:
Yes, that was intentional. Perhaps I should have added a comment there. It is because of two reasons:

1. We do not deal with tables for this specific API call and hence will not have the type anywhere. 
2. The convention here is a bit misleading(I later noticed your issue) and might need to be changed.
(ie: database points to type of database, subtype in our scenario - hive_table and schema points to the database name.)

I am happy to open another PR on this if required. Feel free to suggest. @verdan :)

Reply:
As a solution, I will mention the supertype `Table` until we incorporate this design in the entity defs. :)